src/channel.ts

```diff
@@ -117,13 +117,49 @@ export default class Channel {
     }
   }
 
+  // Disabled channels won't receive automatic updates
```

Comment:
Won't they though? Shouldn't the disabled channels be filtered out when sending messages?

Reply:
Currently it's done in the sendMessage level. We can bubble the check up to avoid excess logging as well I guess

Reply:
I think I'd rather do that in a couple of weeks after we've made sure this is not picking up and false positives.